docker/xgo-image-deb6/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+docker build -t tudorg/xgo-deb6-base base/ && \
```

Comment:
Could you add `--rm=true` to the build commands? this will remove the intermediate containers after a good build to save diskspace.